modules/es-extensions/store/extensions/app/greg-store-defaults/asset.js

```diff
@@ -106,6 +106,11 @@ asset.renderer = function(ctx) {
 
                 var type = page.assets.type;
                 page.assetVersions = gregAPI.getAssetVersions(ctx.session, ctx.assetType, page.assets.path, page.assets.name);
+                var single_version = false;
+                if(page.assetVersions.length === 1){
```

Comment:
Thanks @daneshk , I have fixed it in [this commit](https://github.com/wso2/product-greg/pull/854/commits/6faf5fe5f797312d71e574912011f10a69d42c37)